nb2workflow/__init__.py

```diff
@@ -1,6 +1,8 @@
 import pkg_resources
+import os
 
 name = "nb2workflow"
+conf_dir = os.path.dirname(__file__)+'/config_dir'
```

Comment:
Do you use it anywhere?

Reply:
nope, leftover of a previous implementation